doc-content/enterprise-only/processes/process-instance-details-con.adoc

```diff
@@ -7,11 +7,11 @@ Click any one of the process instances and view the corresponding details.
 
 The process instance page provides several tabs with runtime information related to the process:
 
-* The *Instance Details* tab: This gives you a quick overview about what is going on inside the process. It displays the current state of the instance and the current activity that is being executed.
+* The *Instance Details* tab: This gives you a quick overview about what is going on inside the process. It displays the current state of the instance and the current activity that is being executed. From a subprocess *Instance Details*, you can click the *Parent Process Instance* field to navigate to the parent process *Instance Details*.
```

Comment:
This provides....
To view the details of the subprocess, the *Parent Process Instance* field and navigate to the parent process *Instance Details* page.